goose3/extractors/title.py

```diff
@@ -43,8 +43,9 @@ def clean_title(self, title):
         elif (self.article.schema and "publisher" in self.article.schema and
               "name" in self.article.schema["publisher"]):
             site_name = self.article.schema["publisher"]["name"]
-            # remove the site name from title
-            title = title.replace(site_name, '').strip()
+            # remove the site name from title if title matches the site name
```

Comment:
If I am reading this correctly, this would only remove the site name from the title if it is identical. Therefore, if it is the same (i.e., title == site name) then we would be reporting an empty title. I do not think this is desired.

Reply:
>it is the same (i.e., title == site name) then we would be reporting an empty title. I do not think this is desired.

Yup, you are correct.

Reply:
I missed that case. 😓